tests/unit/onyxCacheTest.js

```diff
@@ -0,0 +1,680 @@
+import React from 'react';
+import {render} from '@testing-library/react-native';
+
+import waitForPromisesToResolve from '../utils/waitForPromisesToResolve';
+import ViewWithText from '../components/ViewWithText';
+import ViewWithCollections from '../components/ViewWithCollections';
+
+describe('Onyx', () => {
+    describe('Cache Service', () => {
+        /** @type OnyxCache */
+        let cache;
+
+        // Always use a "fresh" instance
```

Comment:
NAB, Is that what requiring again would do? When would this not be creating a "fresh" instance? Why should we use one instead of just clearing out the cache?

Reply:
> NAB, Is that what requiring again would do? When would this not be creating a "fresh" instance? Why should we use one instead of just clearing out the cache?

Simply requiring again would not do the trick. It would return the instance that was already created from the first time the script was required somewhere

There's no function that completely clears the cache. You could implemented something here or in the cache file, but if it's only for the test you could just tell jest you want a clean env